scripts/findChrome.js

```diff
@@ -1,3 +1,5 @@
-const { findChrome} = require('./chromeDetection')
+if (!process.env.ESTIMO_DISABLE) process.exit()
```

Comment:
If ESTIMO_DISABLE is true you don't want to execute findChrome. Am i right?
If so, why you are using here negative operator?